project/lvgl/templates/stm32f746g-discovery/mods.conf

```diff
@@ -98,4 +98,6 @@ configuration conf {
 	include third_party.lib.lvgl_display_port_stm32(log_level=4)
 	include third_party.lib.lvgl_input_dev_port(log_level=4)
 	include project.lvgl.cmd.lvgl_demo
+
+	include third_party.cmd.mosquitto
```

Comment:
I think it's better to create a new template in 'project/mosquitto/templates' folder

Reply:
Created